lib/webmock/request_execution_verifier.rb

```diff
@@ -34,26 +34,26 @@ def does_not_match?
     end
 
     def description
-      "request #{request_pattern.to_s} #{quantity_phrase.strip}"
+      "request #{request_pattern} #{quantity_phrase.strip}"
     end
 
     def failure_message
-      failure_message_phrase(is_negated=false)
+      failure_message_phrase(_is_negated = false)
```

Comment:
Hi @rochefort, may ask  question, what you think it is purpose even this unused ``_is_negated`` variable here?
Wouldn't be better if ``is_negated`` was argument of ``failure_message``? Something like:
```ruby
def failure_message(is_negated = false)
  failure_message_phrase(is_negated)
end
```
which would fix the warning as well or by using keywords arguments:
```ruby
def failure_message
  failure_message_phrase(is_negated: false)
end
...

def failure_message_phrase(options = {})
  negation = options.fetch(:is_negated, false) ? "was not" : "was"
...
```